iOS/Flows/Home/Moments/Capture/PIP/PiPRecorder.swift

```diff
@@ -143,13 +141,15 @@ class PiPRecorder {
     private func initializeFront() {
         // Create an asset writer that records to a temporary file
         let outputFileName = NSUUID().uuidString + "front"
-        let outputFileURL = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent(outputFileName).appendingPathExtension("mov")
```

Comment:
Everything else is just formatting changes I made while trying to track down the bug.